src/app/templates/templates-routes.module.ts

```diff
@@ -0,0 +1,17 @@
+import { NgModule } from '@angular/core';
+import { RouterModule, Routes } from '@angular/router';
+
+import { TemplatesModule } from './templates.module';
+import { TemplatesListPage } from './list-page/list-page.component';
+
+const routes: Routes = [
+  { path: '', component: TemplatesListPage, pathMatch: 'full' },
+];
+
+@NgModule({
+  imports: [
+    RouterModule.forChild(routes),
+    TemplatesModule,
+  ],
+})
+export class TemplateRoutesModule { }
```

Comment:
+1 @gashcrumb I prefer `template.routes.ts` or `templateRoutes.module.ts`.. but then again I'm a stickler for camelCase over snake-case. If there will only be template routes we should stick with `routes.module.ts`.